app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    total = 0
+
+    for player in players:
+        total += player.get_rating()
+
+    return total
+
+
+def elves_concert(elves: list[Elf]) -> None:
+    for elf in elves:
```

Comment:
Check prev comment

Reply:
Here's a comment of a mentor - 
![NVIDIA_Share_c7YnjcV1c6](https://github.com/mate-academy/py-elves-and-dwarves/assets/13889031/b49247bf-0c6c-42fd-906d-1d15d43fccad)

Here's a comment from chatGPT when I asked about list comprehension and sum, says sum is good, but comprehension may lead to bad readability

![NVIDIA_Share_oqkuOBD1of](https://github.com/mate-academy/py-elves-and-dwarves/assets/13889031/0b30094b-0b2f-4b49-b442-76d4d44f33c0)